docs/user_guide/source/installation/cmake.rst

```diff
@@ -1,312 +1,20 @@
-**************************************************
-Building, Testing and Installing ADIOS2 with CMake
```

Comment:
We can only build with `CMake` so we might as well say it up front. What would be good to add, if it's not there, is that we only work with `CMake v3.6` and above.

Reply:
@williamfgc : Your `CMakeLists.txt` already has the `cmake_minimum_required(VERSION 3.6)`; you've heard me say it before: intelligible error messages are the best form of documentation so  you've already covered this point.